python/lsst/ts/m2com/utility.py

```diff
@@ -237,12 +238,14 @@ def get_config_dir(
     return Path(getenv(env_variable, default="")) / relative_path
 
 
-def is_coroutine(function: typing.Callable | typing.Coroutine) -> bool:
+def is_coroutine(
+    function: typing.Callable | typing.Coroutine | asyncio.Task | asyncio.Future,
```

Comment:
I think `function` could be of `Any` type no? this method will simple return a boolean if it is a coroutine or not.

Reply:
Updated in the commit: [26c5121](https://github.com/lsst-ts/ts_m2com/pull/48/commits/26c5121742d6dc7696c53a2e3574bd45ad40ad2d). Thanks!